assets/javascripts/discourse/components/webinar-register.js.es6

```diff
@@ -66,6 +66,45 @@ export default Component.extend({
       .finally(() => this.set("loading", false));
   },
 
+  @discourseComputed("webinar.{title}")
+  downloadName(webinar) {
+    return webinar.title + ".ics";
+  },
+
+  @discourseComputed("webinar.{starts_at,ends_at}")
+  addToGoogleCalendarUrl(webinar) {
+    return `http://www.google.com/calendar/event?action=TEMPLATE&text=${encodeURIComponent(
+      webinar.title
+    )}&dates=${this.formatDateForGoogleApi(
+      webinar.starts_at
+    )}/${this.formatDateForGoogleApi(webinar.ends_at)}`;
+  },
+
+  @discourseComputed("webinar.{starts_at,ends_at}")
```

Comment:
Minor, but you likely don't need to compute this on `starts_at` or `ends_at`... in any case, they won't change often.